books/free-programming-books-subjects.md

```diff
@@ -404,7 +404,7 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [The Python Game Book](https://web.archive.org/web/20210308080726/https://thepythongamebook.com/en%3Astart) - Horst Jens *(:card_file_box: archived)*
 * [Top 10 Machine Learning Algorithms Every Engineer Should Know](https://www.dezyre.com/article/top-10-machine-learning-algorithms/202) - Binny Mathews, Omair Aasim
 * [Understanding Machine Learning: From Theory to Algorithms](https://www.cs.huji.ac.il/~shais/UnderstandingMachineLearning) - Shai Shalev-Shwartz, Shai Ben-David
-
+* [Python Data Science Handbook: Essentials Tools for Working with Data](https://jakevdp.github.io/PythonDataScienceHandbook/) - Jake Vanderplas
```

Comment:
This is already added [here](https://github.com/EbookFoundation/free-programming-books/blob/main/books/free-programming-books-langs.md#:~:text=Python%20Data%20Science%20Handbook%20%2D%20Jake%20VanderPlas%20(HTML%2C%20Jupyter%20Notebooks)) 

You can update the PR if you have any other good resource or you can close it